src/DatePicker/Calendar.js

```diff
@@ -46,7 +45,7 @@ class Calendar extends Component {
     open: PropTypes.bool,
     shouldDisableDate: PropTypes.func,
     showActionButtons: PropTypes.bool,
-    wordings: deprecated(PropTypes.object, 'Instead, use `cancelLabel` and `okLabel`.'),
+    wordings: PropTypes.object,
```

Comment:
@mbrookes asked me to just keep it in the Datepicker and remove em from the child components.